test/isMobilePhone.js

```diff
@@ -0,0 +1,11 @@
+import assert from 'assert';
```

Comment:
nit: add the appropriate tests for the new regex, can see previous PRs as @WikiRik had suggested.